src/sdk.ts

```diff
@@ -135,14 +135,42 @@ s	 * @example
 	 *
 	 * @example
 	 * const payments = await client.getSubscriptionPayments();
+	 * const payments = await client.getSubscriptionPayments({ subscriptionID: 123 });
+	 * Legacy usage for backwards compatibility: pass planID as a number instead of options object
 	 * const payments = await client.getSubscriptionPayments(123);
 	 */
-	getSubscriptionPayments(planID?: number) {
+	getSubscriptionPayments(options?: number | {
+		planID?: number;
+		subscriptionID?: number;
+		isPaid?: boolean;
+		from?: Date;
+		to?: Date;
+		isOneOffCharge?: boolean;
+	}) {
+		const opts = (typeof options === 'number') ? { planID: options } : options;
```

Comment:
Let's print a deprecation warning when we receive a `number` here. We'll remove support for this in the name major version.

Reply:
Done in https://github.com/avaly/paddle-sdk/pull/82/commits/a99cac44504df95650f0a0b4332f88d9804fd482